SUPPORTED_LANGUAGES.md

```diff
@@ -17,6 +17,7 @@ Languages that listed a **Package** below are 3rd party languages and are not bu
 | AVR assembler           | avrasm                 |         |
 | ActionScript            | actionscript, as       |         |
 | Alan                    | alan, i                | [highlightjs-alan](https://github.com/highlightjs/highlightjs-alan) |
```

Comment:
Could we change this to "Alan IF" then I think?

Reply:
Certainly, we didn't want to presume since the other Alan has its syntax highlighter hosted by this project, so we had left it as-is.

Reply:
Well see how it goes.